### PR TITLE
fixed setup kubelet docs, which caused kubelet startup exception

### PR DESCRIPTION
**Pull request by JasonRD**
### Ⅱ. Does this pull request fix one issue?

fixed issue https://github.com/koordinator-sh/koordinator/issues/273

### CODE REVIEW

**Review by jasonliu747 — COMMENTED** [review]
/lgtm
🚀🚀🚀

**Comment by JasonRD** [comment]
> /lgtm
> 🚀🚀🚀

hhh, seems to be not automatic merged.

File: docs/design-archive/koord-runtime-proxy-design-doc.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ TODO
 Under containerd scenario, to make koord-runtime-proxy a proxy between kubelet and containerd, kubelet parameters should be altered as shown
 below:
 ```
-kubelet <other options> --container-runtime=remote --container-runtime=/var/run/koord-runtimeproxy/runtimeproxy.sock
+kubelet <other options> --container-runtime=remote --container-runtime-endpoint=/var/run/koord-runtimeproxy/runtimeproxy.sock
```
Comment (eahydra) [inline docs/design-archive/koord-runtime-proxy-design-doc.md:132]: hey,  which version introduced the container-runtime-endpoint parameter?

Comment (jasonliu747) [reply docs/design-archive/koord-runtime-proxy-design-doc.md:132]: > hey, which version introduced the container-runtime-endpoint parameter?

this flag was introduced all the wayyyyyy back to 2017

Comment (eahydra) [reply docs/design-archive/koord-runtime-proxy-design-doc.md:132]: OK… my mistake

**Pull request merged**